qframelesswindow/utils/win32_utils.py

```diff
@@ -1,4 +1,5 @@
 # coding:utf-8
+import ctypes
```

Comment:
We have imported windll, so don't need to import ctypes

Reply:
Just removed